homeassistant/helpers/entity.py

```diff
@@ -58,7 +58,11 @@
 FLOAT_PRECISION = abs(int(math.floor(math.log10(abs(sys.float_info.epsilon))))) - 1
 
 
-ENTITY_CATEGORIES_SCHEMA: Final = vol.In(ENTITY_CATEGORIES)
+# pylint: disable-next=invalid-name
+def ENTITY_CATEGORIES_SCHEMA(value: Any | None) -> EntityCategory:
```

Comment:
Are we worried about changing the name will break custom integrations? I'd suggest we rename it to proper PEP8 as the usage is low in core and probably low elsewhere too.

Reply:
for the hot fix, let's keep the name as is.